example/js/controllers/docCtrl.js

```diff
@@ -1,34 +1,50 @@
-app.controller('DocCtrl', function($scope) {
+app.controller('DocCtrl', function($scope, NgPdfFactory) {
 
   $scope.pdfName = 'Relativity: The Special and General Theory by Albert Einstein';
-  $scope.pdfUrl = 'pdf/relativity.pdf';
-  $scope.pdfPassword = 'test';
   $scope.scroll = 0;
   $scope.loading = 'loading';
+  $scope.changePdfPage = 1;
+
+  $scope.pdfConfig = new NgPdfFactory('pdf/relativity.pdf', { fitToPage: true });
+  $scope.protectedPdfConfig = new NgPdfFactory('pdf/relativity.protected.pdf', { fitToPage: true, useCredentials: true, password: 'test' });
+
+  $scope.$watch('pdfUrl', (newVal, oldVal) => {
```

Comment:
Tests are broken on PhantomJS with ES6 sintax, maybe we can try to update PhantomJS or use old ES5 sintax?